src/index.html

```diff
@@ -24,28 +24,25 @@
   <script src="http://ajax.googleapis.com/ajax/libs/angularjs/1.4.8/angular.min.js"></script>
   <script src="http://ajax.googleapis.com/ajax/libs/angularjs/1.4.8/angular-sanitize.js"></script> 
   
+
+<!-- For the Date Range Picker -->
+<script type="text/javascript" src="//cdn.jsdelivr.net/jquery/1/jquery.min.js"></script>
```

Comment:
jquery is already included. Please remove either this one or the other jquery script tag. Either way, make sure you test.